chains/tendermint/prover.go

```diff
@@ -54,6 +54,11 @@ func (pr *Prover) ProveState(ctx core.QueryContext, path string, value []byte) (
 	}
 }
 
+// ProveHostConsensusState returns the existen proof of the consensus state at `height`
+func (pr *Prover) ProveHostConsensusState(ctx core.QueryContext, height ibcexported.Height, consensusState ibcexported.ConsensusState) ([]byte, error) {
+	return clienttypes.MarshalConsensusState(pr.chain.Codec(), consensusState)
```

Comment:
Since ibc-go ignores `hostConsensusStateProof`, is it also ok to return `nil, nil` here?

Reply:
Fix it and force pushed at https://github.com/hyperledger-labs/yui-relayer/pull/132/commits/a548019459ac42abc18a8588225209411f1c28c0